WORKSPACE

```diff
@@ -88,6 +86,9 @@ ts_setup_workspace()
 load("@angular//:index.bzl", "ng_setup_workspace")
 ng_setup_workspace()
 
+load("@angular_material//:index.bzl", "mat_setup_workspace")
+mat_setup_workspace()
```

Comment:
I would prefer `angular_material_setup_workspace`

Reply:
I don't feel strongly either way but using `mat_setup_workspace` would be more consistent with what is done elsewhere: `ng_setup_workspace` for angular and `ts_setup_workspace` for rules_typescript.

Reply:
My reason for being more specific is that there are other Material Design libraries out there, so generally we make sure that we are always clear about being _Angular_ Material. We also wouldn't want to do `ng_material` because that name was associated with AngularJS Material.

Reply:
Sounds good 👍